packages/opentelemetry-remote-sampler-aws-xray/src/remote-sampler.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Sampler, SamplingDecision, SamplingResult } from '@opentelemetry/sdk-trace-base';
+import { SamplingRule } from './remote-sampler.types';
+import axios from 'axios';
```

Comment:
Should we be using axios instead of the native Node HTTP client? Is this inline with other OTel packages that make HTTP calls?

Reply:
@blumamir looping you in for this discussion as well. I was looking into replacing `axios` with `node-fetch` but I noticed that most packages in this repo are using `axios` and couldn't find any using `node-fetch`. Is there any recommendation for using either one?

Reply:
I don't see any harm in using axios. I know they were avoided in the exporters in favor of the plain HTTP interface, but I think they are ok for higher-level and 3rd party node components.